source/playbooks/NIST80053/lib/security_controls_playbook-construct.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Let's rename this file and the classes within it to reflect that it's for NIST 800-53.